hack/test-cmd.sh

```diff
@@ -337,12 +330,19 @@ for test in "${tests[@]}"; do
   os::cmd::try_until_text "oc get projects -o name" "project/${namespace}"
   os::test::junit::declare_suite_end
 
-  ${test}
+  if ! ${test}; then
```

Comment:
Running the test in an `if` block means we will no longer call any EXIT trap handler that the test may have set up.  Typically, a test sets an EXIT trap handler that invokes `os::test::junit::reconcile_output`, which invokes `os::test::junit::declare_suite_end` as necessary.  Will this change not cause us to fail to ensure we always call `os::test::junit::declare_suite_end` after a failed test, or will our failing to do so not cause problems?